packages/govtnz-ds-website/src/components/ColorSwatch.tsx

```diff
@@ -1,24 +1,27 @@
-import React from 'react'
-import P from '@govtnz/ds/build/react-ts/P'
-import './color-swatch.scss'
+import React from 'react';
+import P from '@govtnz/ds/build/react-ts/P';
+import './color-swatch.scss';
 
 type Props = {
-  colorCode: string
-  colorName: string
-  description?: string
-}
+  colorCode: string;
+  colorName: string;
+  description?: string;
+};
 
 const ColorSwatch = ({ colorCode, colorName, description }: Props) => (
   <div className="color-swatch">
-    <div
-      className={`color-swatch__color ${colorName === 'White' &&
-        'color-swatch__color--border'}`}
-      style={{ backgroundColor: colorCode }}
-    />
+    <svg
```

Comment:
Switching SVG because IE 11 does not print empty `<div>`s by default.

Reply:
Nice fix